stackslib/src/net/tests/inv/nakamoto.rs

```diff
@@ -434,6 +451,50 @@ fn check_inv_messages(
     }
 }
 
+fn check_inv_state(
+    bitvecs: Vec<Vec<bool>>,
+    rc_len: u32,
+    nakamoto_start_burn_height: u64,
+    inv_state: &NakamotoTenureInv,
+) {
+    for (i, (tenure_rc, tenure_inv)) in inv_state.tenures_inv.iter().enumerate() {
+        for bit in 0..(rc_len as usize) {
+            let msg_bit = if bit / 8 >= tenure_inv.len() {
+                // only allowed at the end
+                debug!(
+                    "bit = {}, tenure_rc = {}, tenure_inv = {:?}",
+                    bit, tenure_rc, &tenure_inv
+                );
+                assert_eq!(i, inv_state.tenures_inv.len() - 1);
+                false
+            } else {
+                tenure_inv[bit / 8] & (1 << (bit % 8)) != 0
```

Comment:
This looks dangerous. As Aaron mentioned earlier, can we use either `stacks_common::BitVec` or the `bitvec` crate (it's already in Cargo.lock)